Resources/config/doctrine/BasePage.orm.xml

```diff
@@ -22,6 +22,7 @@
         <field name="templateCode" type="string" column="template" nullable="false"/>
         <field name="createdAt" type="datetime" column="created_at"/>
         <field name="updatedAt" type="datetime" column="updated_at"/>
+        <field name="targetRedirectType" type="integer" column="target_redirect_type" nullable="true" default="302"/>
```

Comment:
@sonata-project/contributors WDYT? Should this be considered a BC-break or not? Solution exist to make this field optional, like a doctrine loadMetadata event listener, but it's more work.